src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,67 @@
+package study;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class StringTest {
+
+    @DisplayName("따옴표를 기준으로 문자열을 슬라이싱 할 수 있다")
+    @Test
+    void splitByComma() {
+        //given
+        String input = "1,2";
+        String delimiter = ",";
+
+        //when
+        String[] result = input.split(delimiter);
+
+        //then
+        assertThat(result).containsExactly("1", "2");
+    }
+
+    @DisplayName("주어진 문자열의 양끝단 괄호들을 제거 할 수 있다")
+    @Test
+    void substring() {
+        //given
+        String input = "(1,2)";
+
+        //when
+        String incompleteResult = input.substring(1);
+        String result = incompleteResult.substring(0, incompleteResult.length() - 1);
```

Comment:
```suggestion
        String result = "(1,2)".substring(1, incompleteResult.length() - 1);
```

입력 값을 굳이 분리하지 않는 것은 어떨까?